graviti/schema/catalog_to_schema.py

```diff
@@ -0,0 +1,322 @@
+
+        if labels:
+            value = labels
+            for _ in range(self._label_value_level):
+                value = value[0]  # type: ignore[assignment]
+            field["items"].update(_get_dtype(value))  # type: ignore[arg-type]
+
+        return field
+
+
+class _ConverterMultiKeypoints2D(
+    _ConverterMultiGeometryLabel
+):  # pylint: disable=too-few-public-methods
+    def __call__(  # type: ignore[override]
+        self, subcatalog: Keypoints2DSubcatalog, labels: List[LabeledKeypoints2D]
+    ) -> Dict[str, Any]:
+        field = super().__call__(subcatalog, labels)  # type: ignore[arg-type]
+
+        items = field["items"]
+        keypoints = subcatalog.keypoints[0]
+        items["number"] = keypoints.number
+        if hasattr(keypoints, "names"):
+            items["names"] = keypoints.names
+        if hasattr(keypoints, "skeleton"):
+            items["skeleton"] = keypoints.skeleton
+        if hasattr(keypoints, "visible"):
+            items["visible"] = keypoints.visible
+
+        return field
+
+
+_ConverterClassification("classification")
+_ConverterMultiGeometryLabel("box2d", "label.Box2D", 2)
+_ConverterMultiLabel("box3d", "label.Box3D")
+_ConverterMultiGeometryLabel("polygon", "label.Polygon", 3)
+_ConverterMultiGeometryLabel("polyline2d", "label.Polyline2D", 3)
+_ConverterMultiGeometryLabel("multi_polygon", "label.MultiPolygon", 4)
+_ConverterMultiGeometryLabel("multi_polyline2d", "label.MultiPolyline2D", 4)
+_ConverterMultiKeypoints2D("keypoints2d", "label.Keypoints2D", 3)
+_ConverterMultiLabel("rle", "label.RLE")
+_ConverterSingleLabel("semantic_mask", "label.file.SemanticMask")
+_ConverterInstanceMask("instance_mask", "label.file.InstanceMask")
+_ConverterPanopticMask("panoptic_mask", "label.file.PanopticMask")
+
+
+def _get_file_field(data: RemoteData, notes: Notes) -> Dict[str, Any]:
+    field = _SUFFIX_TO_FILE_FIELDS.get(Path(data.path).suffix, {"name": "file", "type": "bytes"})
+
+    if field["type"] == "file.PointCloudBin":
+        bin_point_cloud_fields = notes.bin_point_cloud_fields
+        if bin_point_cloud_fields:
+            field["field"] = bin_point_cloud_fields
+
+    if notes.is_continuous:
+        field["isContinuous"] = True
+
+    return field
+
+
+def _get_label_fields(
+    catalog: Catalog, label: Label
+) -> Tuple[List[Dict[str, Any]], List[Dict[str, Any]]]:
+    label_fields: List[Dict[str, Any]] = []
+    label_import_types: List[Dict[str, Any]] = []
+    for label_type, processor in _LABEL_PROCESSORS.items():
+        if not hasattr(catalog, label_type):
+            continue
+        subcatalog = getattr(catalog, label_type)
+        if label_type == "classification":
+            label_fields.extend(processor(subcatalog))
+            label_import_types.extend(processor.import_types)
+        else:
+            label_fields.append(
+                processor(subcatalog, getattr(label, label_type))  # type: ignore[call-arg]
```

Comment:
why ignore this?

Reply:
Because there is only one argument of `_ConverterBase.__call__`.